Code/GraphMol/SubstanceGroup.h

```diff
@@ -82,8 +82,9 @@ class RDKIT_GRAPHMOL_EXPORT SubstanceGroup : public RDProps {
     }
   };
 
+  // Unfortunately, SWIG generated wrapper code uses temporary variables that require a default ctor not be deleted.
   //! No default constructor
-  SubstanceGroup() = delete;
+  // SubstanceGroup() = delete;
```

Comment:
You might be able to use #ifndef SWIG here

Reply:
agreed that this should only be removed for SWIG builds